core/src/main/res/values/strings.xml

```diff
@@ -295,6 +295,7 @@
   <string name="all_files_permission_needed_message">In order to access all the zim files across device we need to have All Files Permission</string>
   <string name="allowed">Allowed</string>
   <string name="not_allowed">Not allowed</string>
+  <string name="android_directory_seperator" translatable="false">/Android</string>
```

Comment:
We are creating the same string resource also in #2800 .

Reply:
those changes are still not merged.

Reply:
those changes are still not merged.